assets/js/lib/settings.js

```diff
@@ -5,10 +5,17 @@ export const EDITOR_FONT_SIZE = {
   large: 16,
 };
 
+export const EDITOR_BACKGROUND_COLOR = {
+  normal: "#282c34",
+  highConstrast: "#060708",
+};
+
 const DEFAULT_SETTINGS = {
   editor_auto_completion: true,
   editor_auto_signature: true,
+  editor_high_contrast: false,
```

Comment:
In the settings we would store the theme, for now either `'default'` or `'high-contrast'`, but we can extend it in the future. And we will keep a checkbox for now, just as we do with font size.